GitExtUtils/FileLogging/LogFilenameCreator.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.IO;
+
+namespace GitExtUtils.FileLogging
+{
+    public class LogFilenameCreator
+    {
+        private readonly string _logName;
+        public readonly string LogPath;
+
+        private Tuple<DateTime, string> _result;
+
+        public LogFilenameCreator(string logPath, string logName)
+        {
+            LogPath = logPath;
+            _logName = logName;
+            _result = Renew();
+        }
+
+        private Tuple<DateTime, string> Renew()
```

Comment:
Please use named tuples.
Also the current implementation is not testable. To make it testable we need to pass the date in:
```suggestion
        private (DateTime Date, string LogFullPath) GetLogFullPath(DateTime date)
```

Reply:
Yes we can inject a date or perhaps have a `IDateTimeProvider` abstraction on the constructor level.